python/tests/test_deserialization.py

```diff
@@ -0,0 +1,279 @@
+from typing import Any
+
+import pytest
+from scylla._rust.session_builder import SessionBuilder  # pyright: ignore[reportMissingModuleSource]
+
+from scylla._rust.results import RowFactory, ColumnIterator  # pyright: ignore[reportMissingModuleSource]
+
+
+@pytest.mark.asyncio
+async def test_simple_deserialization():
+    # 1. Connect
+    builder = SessionBuilder(["127.0.0.2"], 9042)
+    session = await builder.connect()
+
+    # 2. Create keyspace & table
+    await session.execute("""
+        CREATE KEYSPACE IF NOT EXISTS testks
+        WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+    """)
```

Comment:
:wrench: Please never use `SimpleStrategy` - just replace it with `NetworkTopologyStrategy`.